src/drivers/serac.cpp

```diff
@@ -158,16 +148,42 @@ int main(int argc, char* argv[])
   solid_solver.setDisplacementBCs(ess_bdr, disp_coef);
   solid_solver.setTractionBCs(trac_bdr, traction_coef);
 
+  // neo-Hookean material parameters
+  double mu, K;
+  inlet->get("nonlinear_solid/mu", mu);  // has default value
+  inlet->get("nonlinear_solid/K", K);    // has default value
+
   // Set the material parameters
   solid_solver.setHyperelasticMaterialParameters(mu, K);
 
-  // Set the linear solver parameters
-  if (gmres_solver == true) {
+  // Solver parameters
+  serac::NonlinearSolverParameters nonlin_params;
+  inlet->get("nonlinear_solid/solver/nonlinear/rel_tol", nonlin_params.rel_tol);          // has default value
+  inlet->get("nonlinear_solid/solver/nonlinear/abs_tol", nonlin_params.abs_tol);          // has default value
+  inlet->get("nonlinear_solid/solver/nonlinear/max_iter", nonlin_params.max_iter);        // has default value
+  inlet->get("nonlinear_solid/solver/nonlinear/print_level", nonlin_params.print_level);  // has default value
+
+  serac::LinearSolverParameters lin_params;
+  inlet->get("nonlinear_solid/solver/linear/rel_tol", lin_params.rel_tol);          // has default value
+  inlet->get("nonlinear_solid/solver/linear/abs_tol", lin_params.abs_tol);          // has default value
+  inlet->get("nonlinear_solid/solver/linear/max_iter", lin_params.max_iter);        // has default value
+  inlet->get("nonlinear_solid/solver/linear/print_level", lin_params.print_level);  // has default value
```

Comment:
Not sure if it belongs in this PR, but maybe it's possible to have a a constructor for `LinearSolverParams` that accepts the relevant Sidre subtree (in this case `nonlinear_solid/solver/linear`)

Reply:
This PR was to create the basic framework and then move forward with a cleaner solution to this.  Providing either inlet to the constructor or using the idea I proposed in my Axom issue to grab a specific Inlet table and add get to that level would be next.